driver/normalizer/annotation.go

```diff
@@ -88,6 +89,21 @@ func loopAnnotate(typ string, mainRole role.Role, roles ...role.Role) Mapping {
 	}), roles...)
 }
 
+func uncomment_bash(s string) (string, error) {
+	if strings.HasPrefix(s, "#") {
```

Comment:
`return strings.TrimPrefix(s, "#"), nil`

Reply:
Separately: Please use `mixedCase` not `snake_case`. https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps